googletest/src/gtest-port.cc

```diff
@@ -146,6 +150,19 @@ size_t GetThreadCount() {
   }
 }
 
+#elif GTEST_OS_AIX
+
+size_t GetThreadCount() {
+  struct procentry64 entry;
```

Comment:
getprocs64 can be used for either execution mode.  It simply provides a size-invariant procentry64 structure.